tools/slicec-cs/src/cs_options.rs

```diff
@@ -18,6 +18,10 @@ pub struct CsOptions {
     #[arg(long = "rpc", value_enum, default_value_t = RpcProvider::IceRpc, ignore_case = true)]
     pub rpc_provider: RpcProvider,
 
+    /// Enable additional compilation information output.
+    #[arg(long)]
+    pub metrics: bool,
```

Comment:
Since it's for build telemetry, I would not use a different term here.

Reply:
We use it for telemetry but in the future it could include additional information. Information that other tooling may want to use such as the language server. As such, I thought keeping the term more neutral makes more sense.

Reply:
All this code is temporary: slicec-cs will be refactored and once this is done, we should move the build telemetry collection to a slicec plugin.

Why not simply output this hash all the time? It's not like anybody executes slicec-cs "by hand".

Reply:
Very true that's a fair point! No point in trying to make this overly future proof.

Reply:
Thinking more about this I actually think we shouldn't just output the hash at this time. We don't fully know all the telemetry we want to collect in this initial version yet. One additional part of the output I was considering adding was if the user is using any `encoding = Slice1` files. This would be included as part of the JSON output then. Im cool with renaming it to telemetry though!